app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =
+console.log('I think the value of x is bigger than 5');

```

Comment:
This is also not correct, because the value of `x` is not set (yet)

Reply:
changed to : "I think the value of x is undefined"